plot/plot_lmbench

```diff
@@ -0,0 +1,51 @@
+#! /bin/bash
+
+plot() {
+data1="$1"
+data2="$2"
```

Comment:
help use local data2, if this is only within the function

Reply:
file submitted as another PR: https://github.com/intel/lkp-tests/pull/162